### ISSUE TITLE
Entering multiline mode on windows....

### ISSUE THREAD

**Issue by melund** [scopatz/xonsh#644]
I just played with xonsh using it's default settings. And I stumbled on a multiline mode issue, which occurs in windows when `$FORCE_POSIX_PATHS=False` (default). 

When autocompleting folders, a backslash is added to the folder name. I think that is reasonable. But it causes xonsh to enter multiline mode when pressing enter. Pressing one extra time then execute the command. 
```
mel@ANYBODY-MEL ~ $ cd Docu<TAB>
mel@ANYBODY-MEL ~ $ cd Documents\
...................
mel@ANYBODY-MEL ~\Documents $
```

I can see a lot of problems here, but maybe we could have 'FORCE_POSIX_PATHS =True' as default. Even on Windows. At least, until it works consistently.

**Comment by gforsyth** [comment]
Urrrgggggg.  I forgot about `\`s being used for paths in windows.  `FORCE_POSIX_PATHS=True` will fix this, as you say, or we can remove the `\` from the multiline triggers if `ON_WINDOWS`

**Comment by gforsyth** [comment]
or something a little more convoluted that checks if the `cmd` at the beginning of the current line of the PTK document is `cd` and if so to not enter multiline mode...

**Comment by melund** [comment]
Fixed by #645
